tests/datasets/test_errors_processor.py

```diff
@@ -377,8 +379,15 @@ def build_result(self, meta: KafkaMessageMetadata) -> Mapping[str, Any]:
             "modules.version": ["1.13.2", "0.2.0", "0.6.0"],
             "transaction_name": "",
             "num_processing_errors": len(self.errors) if self.errors is not None else 0,
+            "features.key": [],
+            "features.value": [],
         }
 
+        if self.features:
+            for feature in self.features:
+                expected_result["features.key"].append(feature["key"])
+                expected_result["features.value"].append(str(feature["value"]).lower())
```

Comment:
Where does the `.lower()` happen in the processor?

Reply:
Unicodify serializes booleans the way JSON does (e.g. `true`).  Calling `str` in python serializes booleans with the starting letter uppercased.  The correct way to handle this is to use `json.dumps` instead of `str`.  I'll fix it.